pos/.gitignore

```diff
@@ -0,0 +1,14 @@
+/KeyValuePairs.class
```

Comment:
Can be simplified to *.class. This will cover any .class file in any subdirectory